mycroft/skills/phillips_hue/test/intent/sample2.intent.json

```diff
@@ -0,0 +1,7 @@
+{
+  "utterance": "turn on the lights",
+  "intent_type": "TurnOnIntent",
+  "intent": {
+    "TurnOnKeyword": "turn on the lights"
+  }
+}
```

Comment:
Please, add two intent tests for each intent to provide more coverage for your vocab.